### PR TITLE
Effect size calculation

### PR DESCRIPTION
**Pull request by gwaybio**
Adding effect size calculation for `nf1_classifier.py` script. Also minor organizational updates.

### CODE REVIEW

File: scripts/nf1_classifier.py [hunk 1]
```diff
@@ -117,178 +120,236 @@ def train_test_error(clf, train_x, train_y, test_x, test_y):
...
+
+    Arguments:
+    :param score_df: pandas DataFrame with decision function scores and status
+
+    Output:
+    Returns the Cohen's d effect size estimate statistic for two status groups
+    """
+    pos_samples = np.array(score_df[score_df['status'] == 1].decision.tolist())
+    neg_samples = np.array(score_df[score_df['status'] == 0].decision.tolist())
+    var_pos = np.sum((pos_samples - np.mean(pos_samples)) ** 2)
```
Comment (dhimmel) [inline scripts/nf1_classifier.py]: Why not use https://docs.python.org/3.4/library/statistics.html#statistics.variance?

Comment (dhimmel) [reply scripts/nf1_classifier.py]: Noting http://nbviewer.jupyter.org/github/AllenDowney/CompStats/blob/master/effect_size.ipynb -- see cohen's d cell if you're to verify implementations.

**Review by dhimmel — COMMENTED** [review]
Reviewed the cohen's d part of this pull request... not the other changes that are out of scope of "effect size calculation", since I don't think those sections need review.

File: scripts/nf1_classifier.py (hunk 1 above)
Comment (gwaybio) [reply scripts/nf1_classifier.py]: nice! That is definitely a more elegant solution. Thanks for point it out!

**Comment by gwaybio** [comment]
@dhimmel thanks for pointing out that resource - I've updated the function with boosted readability!

**Review by dhimmel — APPROVED** [review]

**Comment by gwaybio** [comment]
@amyecampbell - i won't rush to merge this one! take your time reviewing if you'd like and let me know what you think.

**Pull request merged**